setup.cfg

```diff
@@ -48,7 +48,7 @@ doctest_optionflags =
     IGNORE_EXCEPTION_DETAIL
 text_file_format = rst
 addopts = --doctest-rst
-xfail_strict = true
+# xfail_strict = true  # behavior change in numpy v1.24
```

Comment:
Ideally, we make the failure numpy-version dependent instead of removing this protection.

Reply:
done